### PR TITLE
Feat: vendors and ally component added

### PR DESCRIPTION
**Pull request by DuvanVilladiego**
Layout [US10](https://trello.com/c/Hm76HRKa)

### CODE REVIEW

**Review by NataliaOrtizMa — COMMENTED** [review]
Remember to use colors as variables

File: src/layout/vendors-ally-static.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=main, initial-scale=1.0">
+    <link rel="stylesheet" href="../../src/layout/dd.css">
+    <title>Document</title>
+</head>
+<body>
+    <!--PROVEEDORES-->
+    <div class="vendors">
+        <p>COMO PROVEEDOR</p>
+        <img src="/dist/assets/get-revenue.png" alt="">
```
Comment (llandkoer) [inline src/layout/vendors-ally-static.html:14]: If you use the assets in that folder, it won't work because it's ignored.

Comment (llandkoer) [reply src/layout/vendors-ally-static.html:14]: > Remember to use colors as variables

Totally. In that way you can write code faster and styling will be easier to do and more structured.

**Review by llandkoer — COMMENTED** [review]
Hey, Duvan!

I see you're not using our agreements in the comment of this PR (I can't see to which US it belongs).

**Review by NataliaOrtizMa — APPROVED** [review]
Well done!

**Review by llandkoer — APPROVED** [review]
Great job!

**Pull request merged**